pkgs/by-name/ko/korrect/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  stdenv,
+  fetchCrate,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "korrect";
+  version = "0.1.1";
+
+  src = fetchCrate {
```

Comment:
Why not use `fetchFromGitLab`? Legit question, not being very well versed into Rust myself.

Reply:
Upstream preferred I fetch from Crate, as he feels that this allows him to treat gitlab tags and releases more like a staging ground for experiments, while crates are his official releases. So no real reason, just upstreams preference.

Reply:
Fair enough 🙂